docker/cyclecloud8/scripts/cyclecloud_install.py

```diff
@@ -386,7 +394,9 @@ def modify_cs_config(options):
     move(tmp_cs_config_file, cs_config_file)
 
     #Ensure that the files are created by the cycleserver service user
-    _catch_sys_error(["chown", "-R", "cycle_server.", cycle_root])
+    #   - Recursive chown is not supported if installing as low-priv cycle_server user
+    #_catch_sys_error(["chown", "-R", "cycle_server.", cycle_root])
+    _catch_sys_error(["chown", "cycle_server.", cs_config_file])
```

Comment:
Let's use "cycle_server:" instead of "cycle_server." - it definitely is not valid on all platforms and is relying on some deprecated behaviour.

Reply:
Good point.